src/app/pages/viewing/responses-list/responses-list.component.ts

```diff
@@ -43,22 +47,39 @@ export class ResponsesListComponent implements OnInit {
     }
   }
 
-  ngOnInit(): void { }
+  ngOnInit(): void {
+    if (this.cookieService.check('viewing-cookie') === false) {
+      this.cookieService.set('viewing-cookie', 'true', 365);
+    }
+
+    this.showNewToViewingValue = this.cookieService.get('viewing-cookie');
+
+    if (this.showNewToViewingValue === 'true') {
+      this.showNewToViewing = true;
+    } else if (this.showNewToViewingValue === 'false') {
+      this.showNewToViewing = false;
+    } else {
+      console.log('Error. Viewing-cookie not found or initialized properly.');
```

Comment:
Why do we have a `console.log()` here?

Reply:
You had that in your New-Cookie condition in app-module

Reply:
Err, couldn't you just initialize a new cookie there if there's an error? In any case, we should not be using console.log at all, instead handling the error or using an Alert.

Reply:
@janakitti that was a mistake lmao, I changed it to `AuthService.displayError()`